From 176b7e66ff81f03fc6d9f50091149da1b4f2366c Mon Sep 17 00:00:00 2001
From: Mitch Luban <mluban@nvidia.com>
Date: Tue, 29 Jan 2013 18:32:39 -0800
Subject: [PATCH 04/10] power: Add POWER_HINT_APP_LAUNCH to power hints

Need POWER_HINT_APP_LAUNCH to boost on app launch.

Bug 1155935
Bug 1206410

Reviewed-on: http://git-master/r/195365
(cherry picked from commit 1ed60c55e9089a97443eae6dde2193efe4b7bacf)

Change-Id: I35f260764151edad27df645384756f9c4527a1e5
Reviewed-on: http://git-master/r/203504
Reviewed-by: Simone Willett <swillett@nvidia.com>
Tested-by: Simone Willett <swillett@nvidia.com>
---
 include/hardware/power.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/hardware/power.h b/include/hardware/power.h
index 743462b..b343fe3 100644
--- a/include/hardware/power.h
+++ b/include/hardware/power.h
@@ -42,7 +42,8 @@ typedef enum {
     POWER_HINT_VSYNC = 0x00000001,
     POWER_HINT_INTERACTION = 0x00000002,
     POWER_HINT_APP_PROFILE = 0x00000003,
-    POWER_HINT_COUNT,
+    POWER_HINT_APP_LAUNCH = 0x00000004,
+    POWER_HINT_COUNT = 0x00000005,
 } power_hint_t;
 
 /*
-- 
1.8.1.5

